tests/performance-test/deploy/qdrouterd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: interconnectedcloud.github.io/v1alpha1
+kind: Qdr
+metadata:
+  name: qdr-test
+  namespace: sa-telemetry
+spec:
+  deploymentPlan:
+      image: docker-registry.default.svc:5000/sa-telemetry/amq-interconnect:1.4-7
+      role: edge
+      size: 1
+      placement: Any
+  addresses:
+    - prefix: closest
+      distribution: closest
+    - prefix: multicast
+      distribution: multicast
+    - prefix: unicast
+      distribution: closest
+    - prefix: exclusive
+      distribution: closest
+    - prefix: broadcast
+      distribution: multicast
+    - prefix: collectd
+      distribution: multicast
+    - prefix: collectd/notify
+      distribution: multicast
+  sslProfiles:
```

Comment:
you don't need to specify a cert here, it should just accept what the interior router is using much like we do on the client side for openstack. There is just an `sslProfile` with a `name: default` and no certificates loaded. Then the connection is setup via SSL, but is implicitly authenticated.

Reply:
I'm not exactly sure how all of that works in the Operator based deployment though, so it's possible that configuration for OpenShift isn't valid. Would need to check, but I suppose since you have access to the cert anyways, then why not?

In the future hopefully we'll get all that security stuff sorted out.

Reply:
You're right. I used this when I was trying to debug something before I understood all that the operator was doing

Reply:
It's asymmetrical authentication so there isn't need for that here, just remnant code I accidently left in